ci/bootstrap.sh

```diff
@@ -5,14 +5,21 @@ cd "$(dirname "$0")/../"
 
 SHARED_CI_DIR="$(pwd)/.shared-ci"
 CLONE_URL="git@github.com:spatialos/gdk-for-unity-shared-ci.git"
+PINNED_SHARED_CI_VERSION=$(cat ./ci/shared-ci.pinned)
 
 # Clone the HEAD of the shared CI repo into ".shared-ci"
 
-if [[ -d "${SHARED_CI_DIR}" ]]; then
-    rm -rf "${SHARED_CI_DIR}"
-fi
+rm -rf "${SHARED_CI_DIR}"
```

Comment:
Why does this no longer check if the directory exists before removing?

Reply:
I guess it doesn't really matter? either it exists and we remove it or it doesn't exist and we simply try to remove it

Reply:
^this was my thinking too

Reply:
I assume the test was there for a reason previously, so I'm not sure why it was removed.

Reply:
there was no reason other than the outcome being same with or without the test

added it back as it is a harmless check anyway (updated blank project PR too)